lib/text/vtt_text_parser.js

```diff
@@ -397,6 +430,10 @@ shaka.text.VttTextParser = class {
       const bold = shaka.text.Cue.fontWeight.BOLD;
       const italic = shaka.text.Cue.fontStyle.ITALIC;
       const underline = shaka.text.Cue.textDecoration.UNDERLINE;
+      const textColor = shaka.text.Cue.defaultTextColor;
+      const textColorKeys = Object.keys(textColor);
+      const bgColor = shaka.text.Cue.defaultTextBackgroundColor;
+      const bgColorKeys = Object.keys(bgColor);
       const tags = element.nodeName.split(/[ .]+/);
       for (const tag of tags) {
         if (styles.has(tag)) {
```

Comment:
Looking at this code now...

This approach will lead to weird things happening in the case of a long series of tags, some of which have custom styles. For example...
```
WEBVTT

STYLE
::cue(.bg_blue) { font-size: 10px; }

00:00:20.000 --> 00:00:40.000
<c.red.bg_blue>Example Text</c>
```
I just tested this, and in this case the custom style on the `bg_blue` tag would lead to the parser forgetting the effects of the `red` tag. This is a use-case we aren't testing currently in our tests, but it could come up in the case of default color tags.

But that's out-of-scope for your CL. I can file a GitHub issue for it.